userspace/libscap/engine/modern_bpf/scap_modern_bpf.c

```diff
@@ -29,41 +29,64 @@ limitations under the License.
 #include <sys/utsname.h>
 #include "ringbuffer/ringbuffer.h"
 
+#define REQUIRED_MAJOR 5
+#define REQUIRED_MINOR 8
+#define REQUIRED_PATCH 0
+
 /*=============================== UTILS ===============================*/
 
-static void update_single_64bit_syscall_of_interest(int ppm_sc, bool interesting)
+/// TODO: in the next future, from `oargs` we should directly receive a table of internal syscall code, not ppm_sc.
+static int32_t populate_64bit_interesting_syscalls_table()
```

Comment:
Uhm same here, there is something strange in all the file probably a wrong rebase :thinking:

Reply:
That is weird though, i had no conflicts :/